test_vector_handlers/README.rst

```diff
@@ -0,0 +1,57 @@
+#######################################
+aws-encryption-sdk test vector handlers
+#######################################
+
+There modules provide handlers to be used to process existing, or generate new,
+test vectors based on test vector manifests.
+
+***************
+Getting Started
+***************
+
+Required Prerequisites
+======================
+
+* Python 2.7 or 3.4+
+* aws-encryption-sdk
+
+Use
+===
+
+In addition to direct use of the library, some CLI tools are provided to simplify
+processing of common test manifest types.
+
+Full Message Encrypt
+--------------------
+
+Used to process an AWS Encryption SDK Full Message Encrypt manifest and produce
+a Full Message Decrypt manifest along with all corresponding plaintexts and ciphertexts.
+
+.. code::
+
+    usage: awses-full-message-encrypt [-h] --output OUTPUT --input INPUT [--human]
```

Comment:
what is awses?  Google tell me it's a client for AWS ElasticSeach.

Reply:
Shorthand for "AWS Encryption SDK" ...I forgot that ElasticSearch uses that too...any suggestions for a better shorthand?

Reply:
i'm not willing to fight to hard for it.  It's an internal tool, so the name collision shouldn't be too bad.